lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1150,7 +1156,7 @@ public function get_item_schema() {
 		$base = $this->get_post_type_base( $this->post_type );
 		$schema = array(
 			'$schema'    => 'http://json-schema.org/draft-04/schema#',
-			'title'      => $base,
+			'title'      => $this->post_type,
```

Comment:
@danielbachhuber this was using `posts` before for the "title", however all our other endpoints use singular names. I presumed this `title` refers to the object name, not plural form, so i changed this to just the post type. Do you see issues with that?